Confus/Confus/MazeGenerator.h

```diff
@@ -1,17 +1,26 @@
 #pragma once
+#include <stack>
+
 #include "Maze.h"
-#include "DFSBacktracking.h"
 namespace Confus
 {
 	class MazeGenerator
 	{
 	public:
-		Maze Maze;
-		DFSBacktracking DFSBackTracker;
+		int Seed;
```

Comment:
If I were to modify this value directly (from outside this class), will it actually change the seed, or will it take the seed passed to the constructor/refillMainMaze method? If not, then it might be better to keep it private